<!-- -->
<launch>
    <node pkg="adeye"
          type="ExperimentA.py"
          name="ExperimentA"
          output="screen"/>
</launch>
